Automattic/jetpack
# Protect: Update product upgrade

#### Description
For the time being, the upgrade offered in the Protect UI will be updated from the Security Bundle to Scan.

#### Changes proposed in this Pull Request:

* Updates all references to the Security Bundle to Scan (Pricing page, footer, and threat list item CTA)

#### Other information:

#### Jetpack product discussion


- 1201069996155224-as-1203041702539338

#### Does this pull request change what data or activity we track or use?


- No

#### Testing instructions:






* Checkout this branch
* Fire up your Jurassic Tube
* Verify that the upgrade offered on the Interstitial page initiates the checkout procedure with Scan as the product
* Remove the Scan product from your cart, terminate the checkout process, and return to the Protect admin page
* Review the footer, and verify that the upgrade prompt initiates the checkout procedure with Scan
* Remove the Scan product from your cart, terminate the checkout process, and return to the Protect admin page
* Review the threats list, expand a threat item, and verify that the CTA initiates the checkout procedure with Scan
* Add an applicable plan (Scan, Security Bundle, or Complete) and ensure all conditional upgrade prompts no longer display
